app/routes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
+
+// Route to UPDATE the config
+router.post('/configs', function (req, res) {
+	req.params.update = config.update(req);
+	console.log(req.params.update);
+	res.send(req.params.update);
+});
+
+// Route to READ the configs
+router.get('/configs', function (req, res) {
+	req.params.fetch = config.get();
+	res.send(req.params.fetch);
```

Comment:
you completely misunderstood me, it may have been due to my phone not cooperating. You do not need `req.params.fetch` just call `res.send();` you already added everything to the res object!